reference/data-node-bootstrapping.md

```diff
@@ -0,0 +1,130 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+In order for an access node to be able to use a data node it is
+necessary to ensure that it is properly setup to work as a data
+node. This is called *bootstrapping* the data node.
+
+When an access node adds a new data node to the distributed database
+it either bootstraps the data node or verifies that the data node is
+properly bootstrapped.
+
+A new data node is added to the access node using the `add_data_node`
+command. The default database to bootstrap on the data node is then
+given by the same name, that is if you are connected to the `my_data`
+database on the access node, it will be `my_data` on the data node as
+well. If you do not want to use the same database on the data node as
+on the access node, this can be overridden using the `database` option
+to the `add_data_node` command.
+
+All connections to the data node are made as the current user on the
+access node and it is necessary that the user exists on the data node
+and that it is properly set up to handle password-less logins from the
+access node. This means using either certificates or password files,
+as described in the [Data Node Authentication Chapter][distdb-auth].
+
+When bootstrapping the data node the access node will:
+
+1. Ensure that there is a database on the data node.
+2. Ensure that the extension is available in the database.
+3. Set metadata on the data node.
+
+## Setting up the database on the data node
+
+First, the access node has to ensure that there is a properly
+configured database on the data node by either bootstrapping a new
+database, or verifying that it is correctly set up, depending on the
+value of the `bootstrap` option to `add_data_node`.
```

Comment:
Will to be correct to be specific that it's a flag?
```suggestion
value of the `bootstrap` flag to `add_data_node`.
```

Reply:
@solugebefola Thanks! I didn't think about compliance with our convention. Just thought it might be good idea to clarify that the values can be only `true` or `false`, which I've seen in docs outside Timescale.